### PR TITLE
Stuck stack takeunder.

### LINKED ISSUE #2029: StuckStack takeunder: Build it!

### PR DESCRIPTION
**Pull request by deadlight**
## Done

 - New homepage takeunder
 - Tidied and simplified the takeunder styling

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- See the new takeunder (right hand) in all viewports
- [copy](https://docs.google.com/document/d/1rTHWfeuZbusH_nMBY2QJmoUSdKYTDQucLu1qYXZd_nU/edit?ts=595662bc#)
- [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/takeunders/07.2017/Exportables?preview=07.2017-takeunders.jpg)

## Issue / Card

Fixes #2029 

## Demo

[demo](http://www.ubuntu.com-stuck-stack.demo.haus/)

### CODE REVIEW

File: templates/takeovers/takeunders/_stuck-stack.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="col-6 p-takeunder" style="background-color: #da2643;">
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-8">
+        <h3>
+          <a href="http://ubunt.eu/FZEqZJ"
+          onclick="dataLayerpush({
```
Comment (anthonydillon) [inline templates/takeovers/takeunders/_stuck-stack.html]: Also the function push on the dataLayer object.
```js
dataLayer.push(...)
```

Comment (anthonydillon) [inline templates/takeovers/takeunders/_stuck-stack.html]: That is for both take unders.

**Comment by frankiegranato** [comment]
looks good to me - as long as joana is happy.

**Comment by deadlight** [comment]
@anthonydillon @joanasa89 - colour updated

**Comment by joanasa89** [comment]
Looks good to me. 👍

**Pull request merged**